regen-network/regen-web
# fix: add condition for private post with token

## Description

https://regennetwork.atlassian.net/browse/APP-246

---

### Author Checklist

I have...


### How to test

1. Create a private post.
2. Copy the 'Share the secrete link'
3. Visit the post and verify the location map is loading.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
